scripts/bash/run_enceladus.sh

```diff
@@ -308,18 +307,27 @@ if [[ -z "$DRY_RUN" ]]; then
         sleep 10
       fi
     fi
-
-    # Log the log location
+    # Log the log locationd
     echo "$CMD_LINE" >> "$TMP_PATH_NAME"
     echo "The log will be saved to $TMP_PATH_NAME"
-
-    # Run the job
-    bash -c "$CMD_LINE 2>&1 | tee -a $TMP_PATH_NAME"
-
-    # Report the log location
-    echo
-    echo "Job has finished. The logs are saved to $TMP_PATH_NAME"
+    # Run the job and return exit status of the last failed command in the subshell pipeline (Issue #893)
+    bash -c "set -o pipefail; $CMD_LINE 2>&1 | tee -a $TMP_PATH_NAME"
+	# Save the exit status of spark submit subshell run
+	EXIT_STATUS="$?"
+	# Test if the command executed successfully
+	if [ $EXIT_STATUS -eq 0 ]; then
+		# Report the log location
+		echo
+		echo "Job has finished. The logs are saved to $TMP_PATH_NAME"
```

Comment:
Logs are saved either way, so I would have `if else` for echo `Passed/Failed` and print the rest the same. And either way call `exit $EXIT_STATUS`